metron-interface/metron-config/src/app/sensors/sensor-parser-config-readonly/sensor-parser-config-readonly.component.spec.ts

```diff
@@ -301,7 +301,7 @@ describe('Component: SensorParserConfigReadonly', () => {
   }));
 
   it('should have metadata defined ', async(() => {
```

Comment:
Why is this '32'?  Maybe a comment for a maintainer down the road?

Reply:
Sure I will add a comment here.